pkg/storaged/client.js

```diff
@@ -804,7 +804,7 @@ function legacy_vdo_overlay() {
             }
 
             const v = {
-                name: name,
+                name,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)